product/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path
+
+from product.views import (
+    MainCategoryView, ProductView,
+    ProducListView, MainProductView)
+
+urlpatterns = [
+    path('', ProducListView.as_view()),
+    path('/mainCategory', MainCategoryView.as_view()),
```

Comment:
```suggestion
    path('/main-category', MainCategoryView.as_view()),
```
url은 단어가 이어질 경우 보통 -로 연결합니다.

Reply:
바로 수정하겠습니다!